pkg/apis/serving/v1alpha2/transformer.go

```diff
@@ -51,10 +51,18 @@ func (t *TransformerSpec) Validate(config *InferenceServicesConfig) error {
 	if err != nil {
 		return err
 	}
-	if err := validateResourceRequirements(&transformer.GetContainerSpec().Resources); err != nil {
-		return err
+	errs := []error{
+		validateReplicas(t.MinReplicas, t.MaxReplicas),
+		validateResourceRequirements(&transformer.GetContainerSpec().Resources),
+		transformer.Validate(config),
+	}
+	for _, err := range errs {
```

Comment:
If you wanted to be extra fancy, you could define errs inline.

`for _, err := range []error {}`

Reply:
Great suggestion. I will do this for all of transformer/explainer/predictor.